release-team/role-handbooks/release-signal/README.md

```diff
@@ -137,18 +138,39 @@ stabilization phase.
   the [teams.yaml](https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-release/teams.yaml).
   _Coordinate with the release lead to make multiple changes to this file in one PR._
 - Plan release support and status reporting. Setup [CI-Signal project board](/release-team/role-handbooks/release-signal/project-boards.md#setting-up-the-ci-signal-project-board) and the [Bug Triage Project Board](/release-team/role-handbooks/release-signal/project-boards.md#setting-up-the-bug-triage-project-board).
-- Create an excel sheet to track various assignments during the release cycle. [Example](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit?usp=sharing)
 - Release Signal Shadows must be a member of the Kubernetes organization and, therefore, apply for membership by opening an issue
   on [kubernetes/org](https://github.com/kubernetes/org) (
   see [Issue template](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.yml&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E)).
 - Organizing an onboarding meeting with shadows to walk through this handbook and useful tools like [Testgrid](https://github.com/GoogleCloudPlatform/testgrid),
   [Triage](https://github.com/kubernetes/test-infra/tree/master/triage), [Spyglass](https://docs.prow.k8s.io/docs/spyglass/), [Prow](https://docs.prow.k8s.io/docs/), and [Tide](https://docs.prow.k8s.io/docs/components/core/tide/).
 
+#### Create Assignment Excel Sheet
+
+Create an excel sheet to track various assignments during the release cycle. [Example](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit?usp=sharing)
+
+Roles in the assignments excel sheet:
+
+1. **Primary Reporter:** The primary reporter assigned to a task is responsible for completing it, such as providing meeting updates, giving Go/No-Go signals, etc.
+2. **Main Coordinator:** Main coordinator is responsible for:
```

Comment:
Looking at this [sheet](https://docs.google.com/spreadsheets/d/1lYbCOGvG9jD-AVT0Ajc3LoSLNKHC6xIIy1t6BJPlO7E/edit#gid=1052529613), I saw there are many weeks when a single person is assigned as both `Primary Reporter` and `Main Coordinator`.

Trying to understand if one person is doing both roles, then what is the significance of 2 roles here?

Reply:
> Trying to understand if one person is doing both roles, then what is the significance of 2 roles here?

Agreed, it should be clearer. The primary Reporter is usually assigned to meetings and release cuts. The Main Coordinator is responsible for internal coordination. I'll open a PR for this.